backoffice/templates/backoffice/user.html

```diff
@@ -162,6 +155,16 @@ <h3>{% trans "Enrollments" %}</h3>
 
         <div class="col-md-12">
 
+            {% if userform.instance.is_active and not userform.instance.is_superuser and not userform.instance.is_staff %}
+                <form action="{% url 'masquerade-mask' %}" method="POST" target="_blank" style="display: inline">
```

Comment:
Can you explain the usage of `target="_blank"` here? Redirecting to `MASQUERADE_REDIRECT_URL` would break admin user workflow?

Are you sure that the `style` attribute is required? You cannot use a CSS class instead?

Reply:
target blank because we can now continue being logged on the backoffice while impersonating a user on the front.
So it's better to keep the backoffice tab open so you can come back to it after you finish looking at the user's account.
You can then easily press the stop impersonating button.

For "style", I just copied what is being done elsewhere on the page.